src/dune/cinaps.ml

```diff
@@ -110,4 +109,10 @@ let gen_rules sctx t ~dir ~scope =
          :: List.map cinapsed_files ~f:(fun fn ->
                 let fn = Path.build fn in
                 A.diff ~optional:true fn
-                  (Path.extend_basename fn ~suffix:".cinaps-corrected")) )))
+                  (Path.extend_basename fn ~suffix:".cinaps-corrected")) ))
+  in
+  Super_context.add_alias_action sctx ~dir ~loc:(Some loc) ~stamp:"cinaps"
```

Comment:
I would suggest to move `Alias.cinaps` to this module. The reason is cinaps is just an extension and I don't think it's worth hardcoding its existence in places where it's not completely necessary.

Reply:
Sure, I'll move it!

Reply:
Should it be considered a "standard" alias then?

It seems that currently it can't be made a standard alias from outside the `Alias` module.

Reply:
`standard` is something that allows the alias not to error when it's empty? is that necessary? I don't think the manually defined `cinaps` alias has that either. Let's see if @diml has a preference.

Reply:
I moved the alias to `Cinaps` and therefore made it a regular alias. I'm happy to move it back to `Alias` or to expose `make_standard` if we want to make it that way!

Reply:
Either way works for me.

Reply:
Making `cinaps` standard would mean that user can call `dune build @cinaps` in a project-independent manner, i.e. without having to care whether `cinaps` is used or not. This is valuable for tests for instance, but a bit less for cinaps given that it is not very widespread. 

When you think about it, it would make sense to include cinaps validation in the linting phase. We could do so by attaching it to the `lint` alias, which is already standard.

Finally, we could also have a general syntax for allowing a recursive alias to be empty. Such as `dune build @?cinaps`.

In conclusion, for now I wouldn't make `cinaps` standard. We can always revisit this decision in the future.